plugins/dev_tools/commands/__init__.py

```diff
@@ -6,6 +9,9 @@
 from .initialize_group import initialize_group
 
 list_commands = [
+    get_tests_of_command,
+    update_test_command_folder,
```

Comment:
Why about putting it in a `tests` subgroup ?
Naming could be clarified, something like
```suggestion
    update_tests_command,
```